### PR TITLE
first hardcore stuff

### PR DESCRIPTION
**Pull request by hornygranny**

### CODE REVIEW

File: code/__DEFINES/preferences.dm [hunk 1]
```diff
@@ -43,4 +43,5 @@
 #define BE_GANG			4096
 #define BE_SHADOWLING	8192
 #define BE_ABDUCTOR		16384
-#define BE_REVENANT		32768
+#define BE_REVENANT		32768
+#define BE_CLONE		65536
```
Comment (RemieRichards) [inline code/__DEFINES/preferences.dm]: isn't this only 16 bit flags?

Comment (hornygranny) [inline code/__DEFINES/preferences.dm]: someone said that the flag could store values that high, but you wouldn't be able to do operations on them. we'll see ayy lmao

Comment (RemieRichards) [inline code/__DEFINES/preferences.dm]: ~worrying~

**Comment by RemieRichards** [comment]
Besides the potential flag issue the rest of this seems interesting, the option to not be cloned, and it seems like cloning will be able to properly grab clients, instead of just pinging the ghost and asking them to get in.

**Comment by Iamgoofball** [comment]
for about 5 seconds I was super hyped because I thought this was my Clone gamemode

i'm sad now

**Comment by Cheridan** [comment]
reverting with 0 discussion

**Comment by Iamgoofball** [comment]
wow cheri ur abusin!!!

ima turn on dezzmont mode and write a platitude in feedback about you

**Pull request merged**